src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int FIGURE_RANDOM_PARAMETR = 20;
+    private static final int DEFAULT_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String figureColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (randomNumber) {
+            case 0:
+                return createCircle(figureColor);
+            case 1:
+                return createSquare(figureColor);
+            case 2:
+                return createRightTriangle(figureColor);
+            case 3:
+                return createRectangle(figureColor);
+            case 4:
+                return createIsoscelesTrapezoid(figureColor);
+            default:
+                return new Circle(10, Color.WHITE.name());
```

Comment:
```suggestion
                return new Circle(DEFAULT_RADIUS , Color.WHITE.name());
```

Reply:
<img width="1792" alt="Screenshot 2024-04-18 at 21 12 43" src="https://github.com/mate-academy/jv-oop-advanced/assets/48392239/aeaf7a28-f985-45c1-9e58-0811c5cffe36">
According to the comments - judging by the class Circle- there are not the latest changes - these are the ones that were - and the method setColor
 was removed